docs/htdocs/info/docs/tools/vep/script/vep_download.html

```diff
@@ -1569,6 +1569,52 @@ <h3>Update from a previous version</h3>
   </li>
 </ol>
 
+
+  <hr/>
+  <h2 id="nextflow">Nextflow</h2>
+
+  <p>We offer a <a href="https://github.com/Ensembl/ensembl-vep/tree/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/nextflow" rel="external">Nextflow VEP pipeline</a> that aims to run VEP using simple parallelisation. The pipeline is deployable on an individual Linux machine or on computing clusters running LSF, SLURM or other workload managers.</p>
+
+  <p>The process can be summarised briefly by the following steps:</p>
+
+  <ul>
+    <li>Splitting the input data into multiple files using a given number of bins</li>
+    <li>Running VEP on the split files in parallel</li>
+    <li>Merging VEP outputs into a single file</li>
+  </ul>
+
+  <p>To run the pipeline in a system with <a href="https://nextflow.io" rel="external">Nexflow</a> installed, you will need to prepare a <a href="vep_options.html#opt_config"><b>vep.ini</b> config file</a>. Here are some examples commands to run the Nextflow VEP pipeline:</p>
+
+  <pre class="code sh_sh">
+# Run Nextflow VEP using local VEP installation
+# NB: Nextflow automatically downloads the GitHub repository
```

Comment:
@likhitha-surapaneni , I am not clear in this line, should we omit it?

Reply:
Nextflow is downloading the ensembl-vep repo automatically without the user explicitly cloning or downloading it.